### ISSUE TITLE
datepicker time resets to 12am after publishing if a field is empty

### ISSUE THREAD

**Issue by montchr** [alleyinteractive/wordpress-fieldmanager#296]
![](http://cl.ly/image/3715362G0s21/Screen%20Recording%202015-01-19%20at%2012%3A13.gif)

And on a fresh install with no other plugins and vanilla Twenty Fifteen (sorry for the quality):

![](http://cl.ly/image/3h2Z2q0g3o3j/Screen%20Recording%202015-01-19%20at%2012%3A30.gif)

[Higher-quality version (MOV)](http://cl.ly/02123v2E3l2q/Screen%20Recording%202015-01-19%20at%2012%3A28.mov).

Here's the code that generates it:

```php
add_action( 'init', function() {
  $fm = new \Fieldmanager_Group( false, array(
  			'name'       => 'event_details',
  			'children'   => array(
  				'start_time'     => new \Fieldmanager_Datepicker( esc_html__( 'Start Time', 'pedestal' ), array(
  					'name'       => 'start_time',
  					'use_time'   => true,
  					) ),
  				'end_time'     => new \Fieldmanager_Datepicker( esc_html__( 'End Time', 'pedestal' ), array(
  					'name'       => 'end_time',
  					'use_time'   => true,
  					) ),
  				'venue_name'     => new \Fieldmanager_Textfield( esc_html__( 'Venue Name', 'pedestal' ), array(
  					'name'       => 'venue_name',
  					) ),
  				'address'     => new \Fieldmanager_Textfield( esc_html__( 'Address', 'pedestal' ), array(
  					'name'       => 'address',
  					) ),
  				'cost'        => new \Fieldmanager_Textfield( esc_html__( 'Cost', 'pedestal' ), array(
  					'name'       => 'cost',
  					) ),
  				),
  			) );
	$fm->add_meta_box( 'Event Stuff', array( 'post' ) );
} );
```

**Comment by montchr** [comment]
To clarify: if I fill in both time fields (hour and minute) for both start and end time, then the times do not reset.

![](http://cl.ly/image/430t0e2i2c1c/Screen%20Recording%202015-01-19%20at%2012%3A52.gif)

**Comment by mboynes** [comment]
Resolved in #297
